docs/data/material/components/text-fields/text-fields.md

```diff
@@ -318,10 +318,12 @@ const MyInputComponent = React.forwardRef((props, ref) => {
 
 // usage
 <TextField
-  InputProps={{
-    inputComponent: MyInputComponent,
-    inputProps: {
-      component: SomeThirdPartyComponent,
+  slotProps={{
+    input: {
+      inputComponent: MyInputComponent,
+      inputProps: {
+        component: SomeThirdPartyComponent,
+      },
```

Comment:
Can you create a CodeSandbox to double check that this works?

Reply:
Done, Does this work ?

https://stackblitz.com/edit/react-lh33kf?file=Demo.tsx